packages/@react-types/table/src/index.d.ts

```diff
@@ -79,9 +79,16 @@ export interface CellProps {
 export type CellElement = ReactElement<CellProps>;
 export type CellRenderer = (columnKey: Key) => CellElement;
 
+// TODO: perhaps defined `rows`, etc here so that we can table specific definitions
```

Comment:
yeesh, dunno what happened to my grammar here. Open question: should I redefined `rows` and some of the other GridCollection props so that they are more table specific?